src/WebJobs.Script/Workers/Profiles/Conditions/HostPropertyCondition.cs

```diff
@@ -32,8 +32,14 @@ public HostPropertyCondition(ILogger logger, ISystemRuntimeInformation systemRun
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _systemRuntimeInformation = systemRuntimeInformation ?? throw new ArgumentNullException(nameof(systemRuntimeInformation));
 
-            descriptor.Properties.TryGetValue(WorkerConstants.WorkerDescriptionProfileConditionName, out _name);
-            descriptor.Properties.TryGetValue(WorkerConstants.WorkerDescriptionProfileConditionExpression, out _expression);
+            if (descriptor.Properties.TryGetValue(WorkerConstants.WorkerDescriptionProfileConditionName, out var conditionNameElement))
+            {
+                _name = conditionNameElement.GetString();
+            }
+            if (descriptor.Properties.TryGetValue(WorkerConstants.WorkerDescriptionProfileConditionExpression, out var conditionExpressionElement))
```

Comment:
Same comment as above; add a line above the second if statement.

Reply:
added line break.

Reply:
We should add this to the stylecop rule set